src/node_os.cc

```diff
@@ -149,7 +154,7 @@ static void GetCPUInfo(const FunctionCallbackInfo<Value>& args) {
 
 
 static void GetFreeMemory(const FunctionCallbackInfo<Value>& args) {
-  double amount = uv_get_free_memory();
+  double amount = static_cast<double>(uv_get_free_memory());
   if (amount < 0)
     return;
```

Comment:
Maybe something for a follow-up commit: uv_get_free_memory() and uv_get_total_memory() return an uint64_t, `amount < 0` is always false.